CHANGES.rst

```diff
@@ -63,6 +63,11 @@
 - Faster algorithm for defining object and background regions
 - Restore armsgs -d functionality
 - Finished cython to python conversions, but more testing needed
+- Introduce maskslits array (could be called maskorders)
```

Comment:
I prefer maskslits ('orders' is very specific to echelle). 'slits' is a more general term. So you can remove the text in brackets here, if you like? Another minor comment: Maybe remove the word "Actually" on like 67?  :-)

Reply:
removed+edited